jobs/types/user_validation_errors.py

```diff
@@ -62,3 +62,12 @@ def __init__(self, model):
         self.message = (
             'draft change list last updated %s is greater than the time '
             'when job was run' % model.draft_change_list_last_updated)
+
+
+class ArchivedModelNotMarkedDeletedError(
+        base_validation_errors.BaseAuditError):
+    """Error class for models which are archived but not deleted."""
+
+    def __init__(self, model):
+        super(ArchivedModelNotMarkedDeletedError, self).__init__(model)
+        self.message = 'mark archived model as deleted'
```

Comment:
This message isn't too helpful, please use the docstring instead:
```suggestion
        self.message = 'model is archived but not marked as deleted'
```

Reply:
done